frontend/src/store/modules/auth.js

```diff
@@ -20,8 +20,9 @@ export default {
         const payload = JSON.parse(atob(token.split('.')[1]));
         const roles = payload.realm_access.roles;
 
+        // TODO: this will require re-conceptualizing as acronyms will come from the DB, not the access roles in the future.
         if (typeof roles === 'object' && roles instanceof Array) {
-          state.acronyms = roles.filter(role => !role.match(/offline_access|uma_authorization/));
+          state.acronyms = roles.filter(role => !role.match(/offline_access|uma_authorization|WEBADE_CFG_READ|WEBADE_CFG_READ_ALL/));
```

Comment:
should this be using utils.filterAppAcronyms?

Reply:
I would, but this instance is in the frontend (and the utils class is in the backend).
Since this is the only instance in the frontend project that does this action I didn't refactor it out and duplicate the util function into the frontend.

Reply:
Ahhh, yes.